client/src/Movies/Movie.js

```diff
@@ -32,26 +35,20 @@ const Movie = (props) => {
 
   const { title, director, metascore, stars } = movie;
   return (
-    <div className="save-wrapper">
-      <div className="movie-card">
-        <h2>{title}</h2>
-        <div className="movie-director">
-          Director: <em>{director}</em>
+    <>
+    <MovieCard title={title} director={director} metascore={metascore} />
+      <div>
+        <div>
+          {stars.map(star => (
+            <div key={star} className="movie-star">
```

Comment:
key values shouldn't be a complex data type such as an object.  It works, but gives the app more work to do to when comparing different elements to see what needs to be rendered